.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['12', '14']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
```

Comment:
`actions/setup-node@v2` is the latest stable version now, so you might want to bump this version. 😉

Reply:
thanks, will upgrade this everywhere

Reply:
Thank you! 😄